### PR TITLE
Download all link

### PR DESCRIPTION
**Pull request by colinbruce**
# What

Add a download all files link for caseworkers

# Why

To allow caseworkers quick downloads of all files, rather than downloading the files individually

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -104,10 +104,9 @@ GEM
       descendants_tracker (~> 0.0.1)
     colorize (0.8.1)
     concurrent-ruby (1.0.5)
-    config (1.5.0)
+    config (1.2.1)
```
Comment (jsugarman) [inline Gemfile.lock]: is this the result of needing a rebase on the gem refresh PR?

File: app/services/s3_zip_downloader.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class S3ZipDownloader
+  require 'zip'
+
+  def initialize(claim)
+    @claim = claim
+    @folder = Dir.mktmpdir("#{@claim.case_number}-", './tmp')
```
Comment (jsugarman) [inline app/services/s3_zip_downloader.rb:6]: cool, perhaps a
```
Rails.root.join('tmp') => Pathname object
```
or
```
File.join(Rails.root, 'tmp') => string
```

Comment (jsugarman) [reply app/services/s3_zip_downloader.rb:6]: but just a suggestion

**Review by jsugarman — APPROVED** [review]
👍 couple of comments

File: Gemfile.lock (hunk 1 above)
Comment (colinbruce) [reply Gemfile.lock]: Hmm Good question... rolling back seems a bad idea... I thought I had rebased, but I'll double check.

File: app/services/s3_zip_downloader.rb (hunk 2 above)
Comment (colinbruce) [reply app/services/s3_zip_downloader.rb:6]: I just used the standard ruby invocation of https://apidock.com/ruby/Dir/mktmpdir/class

**Pull request merged**